src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,10 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user).map(User::getAge)
```

Comment:
```suggestion
        return Optional.ofNullable(user)
                .map(User::getAge)
```
한 줄에 하나의 스트림 메소드를 볼 수 있도록 해주는게 가독성에 좋습니다!